README.adoc

```diff
@@ -719,37 +719,37 @@ Eu fico feliz apenas que ela não sofreu abuso físico. Eu acho fascinante como
 
 March 2015: 15 days in jail for no reason: https://www.facebook.com/cirosantilli/posts/952661734753174
 
-June 2017: 3 cops came to her house. She was there. They asked if she still did Falun Gong. She said yes. They took photos of her Falun Gong books / posters. They were polite.
+June 2017: 3 cops came to her house. She was there. They asked if she still did Falun Gong. She said yes. They took photos of her Falun Gong books/posters. They were polite.
 
-October 2017: 7 - 8 cops came to her house _at 11PM_. They knocked the door strongly and made noise, and questioned neighbours of her whereabouts. Luckily she was not there.
+October 2017: 7 - 8 cops came to her house _at 11 PM_. They knocked on the door strongly and made noise, and questioned neighbors of her whereabouts. Luckily she was not there.
 
 ==== Are you a radical?
 
-I don't consider myself a radical because of: <<preconceived>>
+I don't consider myself a radical because: <<preconceived>>
 
-I never get mad. Only a slightly sad or annoyed sometimes.
+I never get mad — only slightly sad or annoyed sometimes.
```

Comment:
Asciidoctor has em-dash with `--`: https://asciidoctor.org/docs/user-manual/#replacements let's use that instead.